### PR TITLE
refactor: change return type to void in RememberModel

### PR DESCRIPTION
**Pull request by kenjis**
- change return type to void when *command* methods 
- and throw Exception if error occurs

### CODE REVIEW

**Review by lonnieezell — APPROVED** [review]

File: src/Models/RememberModel.php [hunk 1]
```diff
@@ -24,25 +25,36 @@ class RememberModel extends Model
...
+
+    private function checkQueryReturn(bool $return): void
+    {
+        if ($return === false) {
+            $error   = $this->db->error();
+            $message = 'Query error: ' . $error['code'] . ', '
+                . $error['message'] . ', query: ' . $this->db->getLastQuery();
+
+            throw new RuntimeException($message, $error['code']);
+        }
```
Comment (MGatner) [inline src/Models/RememberModel.php:51]: The similarities here make me wonder if we should consider a central error handling library. Maybe for the next version? I know Lonnie had a very nice PR on the main repo expanding and centralizing exception handling that might be applicable.

File: src/Models/RememberModel.php [hunk 2]
```diff
@@ -24,25 +25,36 @@ class RememberModel extends Model
...
     }
 
     /**
      * Returns the remember-me token info for a given selector.
      *
-     * @return mixed
+     * @return stdClass|null
      */
     public function getRememberToken(string $selector)
```
Comment (MGatner) [inline src/Models/RememberModel.php:59]: Can this be official?
```php
... $selector): ?stdClass
```

**Comment by MGatner** [comment]
I didn't realize this one was merge already 🤦‍♂️

File: src/Models/RememberModel.php (hunk 1 above)
Comment (kenjis) [reply src/Models/RememberModel.php:51]: I think it is not good that the behavior of the database class changes depending on the value of `DBDebug`.
It should always throw an Exception on database errors.

Comment (MGatner) [reply src/Models/RememberModel.php:51]: Agree! But as you know from recent conversations, I'm much more into throwing exceptions 🤗

**Pull request merged**